src/runtime/eng/RuntimeIdentifier.props

```diff
@@ -23,6 +23,8 @@
     <!-- On Windows, we can build for Windows and Mobile.
          For other TargetOSes, create a "win" build, built from TargetOS sources and "win" pre-built packages. -->
     <PortableOS Condition="'$(HostOS)' == 'win' and '$(TargetsMobile)' != 'true'">win</PortableOS>
+
+    <PortableTargetRid Condition="'$(PortableTargetRid)' == ''">$(PortableOS)-$(TargetArchitecture)</PortableTargetRid>
```

Comment:
This should be a good place to initialize `PortableTargetRid` when `runtime` gets built stand-alone.

Reply:
@ViktorHofer would it be meaningful to make a PR against runtime repo to validate the runtime changes in this PR?